### PR TITLE
Add TypeScript type definition

### PR DESCRIPTION
**Pull request by byeokim**
TypeScript type definition is added.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "node": ">=8.9.0"
   },
   "main": "dist/raw.macro.js",
+  "types": "raw.macro.d.ts",
   "license": "MIT",
   "files": [
     "dist/"
```
Comment (pveyes) [inline package.json]: You'll need to add the file to this property as well

**Review by pveyes — COMMENTED** [review]
Nice! Thanks for making this PR. Fix 1 issue and it's good to go

File: package.json (hunk 1 above)
Comment (byeokim) [reply package.json]: Thank you for guidance! By the way I don't know why other lines in my later commit were prettier-ed.

Comment (pveyes) [reply package.json]: nah it's fine

**Review by pveyes — APPROVED** [review]
Thank you

**Comment by pveyes** [comment]
published as 0.3.1

**Pull request merged**